kafka_consumer/tests/test_unit.py

```diff
@@ -260,18 +260,18 @@ def test_when_highwater_metric_count_hit_context_limit_then_no_more_highwater_me
     # highwater_offset = {(topic, partition): offset}
     highwater_offset = {("topic1", "partition1"): 3, ("topic2", "partition2"): 3}
     mock_client = mock.MagicMock()
-    mock_client.get_consumer_offsets_dict.return_value = consumer_offset
-    mock_client.get_highwater_offsets_dict.return_value = highwater_offset
+    mock_client.get_consumer_offsets.return_value = consumer_offset
+    mock_client.get_highwater_offsets.return_value = highwater_offset
     mock_client.get_partitions_for_topic.return_value = ['partition1']
     mock_generic_client.return_value = mock_client
     caplog.set_level(logging.WARNING)
 
     # When
-    kafka_consumer_check = KafkaCheck('kafka_consumer', {'max_partition_contexts': 1}, [kafka_instance])
+    kafka_consumer_check = KafkaCheck('kafka_consumer', {'max_partition_contexts': 2}, [kafka_instance])
     dd_run_check(kafka_consumer_check)
 
     # Then
-    aggregator.assert_metric("kafka.broker_offset", count=1)
+    aggregator.assert_metric("kafka.broker_offset", count=2)
```

Comment:
Ah that is a good catch! I should have made an assert here to verify that `get_highwater_offsets()` was actually called in the first place rather than just returning a mocked `get_highwater_offsets_dict()`.